### PR TITLE
[CHK-5870] Add setting to toggle express pay visibility

### PR DESCRIPTION
**Pull request by nicolenorman**
Adds a setting in the Magento admin to toggle the visibility of the express pay option at the top of checkout. 

Also adds entries to the csp_whitelist file for ease of local development

image

### CODE REVIEW

**Comment by nicolenorman** [comment]
@JosephLeedy Created this PR to merge into the CHK-5237 branch since it's based on our active work. Happy to edit this to merge into `express-pay` or `main` branch once your piece is wrapped up and merged, just want to make sure this setting goes out with the express pay feature

**Comment by JosephLeedy** [comment]
> @JosephLeedy Created this PR to merge into the CHK-5237 branch since it's based on our active work. Happy to edit this to merge into `express-pay` or `main` branch once your piece is wrapped up and merged, just want to make sure this setting goes out with the express pay feature

Could you please base this off of the Express Pay to avoid issues or conflicts?

**Comment by nicolenorman** [comment]
> > @JosephLeedy Created this PR to merge into the CHK-5237 branch since it's based on our active work. Happy to edit this to merge into `express-pay` or `main` branch once your piece is wrapped up and merged, just want to make sure this setting goes out with the express pay feature
> 
> Could you please base this off of the Express Pay to avoid issues or conflicts?

Going to mark this as draft for now. This PR includes changes inside the `bold-express-pay.js` view file which has been renamed and drastically changed between the `express-pay` head and both of our latest branches. It will be easier to rebase this on top of `express-pay` after the other changes get merged.

**Comment by nicolenorman** [comment]
@JosephLeedy this is now re-based on `express-pay` and ready for review 🚀

**Review by bold-eric-d — APPROVED** [review]

**Review by bCWarkent — APPROVED** [review]

File: Model/Config.php [hunk 1]
```diff
@@ -301,4 +302,19 @@ public function getConfigurationGroupLabel(int $websiteId): ?string
...
+
+    /**
+     * Check if Express Pay buttons are enabled.
+     *
+     * @param int $websiteId
+     * @return bool
+     */
+    public function isExpressPayEnabled(int $websiteId): bool
+    {
+        return (bool)$this->scopeConfig->getValue(
```
Comment (JosephLeedy) [inline Model/Config.php:314]: Since you're retrieving a boolean value here, you _should_ use `isSetFlag()` to get the value rather than using `getValue()` and casting the result to boolean.

File: etc/csp_whitelist.xml
Comment (JosephLeedy) [inline etc/csp_whitelist.xml:1]: These change should be ideally be part of separate PR, but I think this is fine just this one time.

**Review by JosephLeedy — CHANGES_REQUESTED** [review]

File: Model/Config.php (hunk 1 above)
Comment (nicolenorman) [reply Model/Config.php]: Good call,  updated.

**Review by JosephLeedy — APPROVED** [review]
